### ISSUE TITLE
Firmware 4.28.17623 (kobo8 / Elipsa)

### ISSUE THREAD

**Issue by pgaskin** [pgaskin/NickelMenu#109]

**Comment by pgaskin** [comment]
All required symbols are present on this version, so existing releases should work correctly. It'd be nice if someone with an Elipsa could test NM to ensure things work properly.

**Comment by pgaskin** [comment]
New action ideas:
- `nickel_open` for notepad, etc.

**Comment by mwt** [comment]
NM does not work for me on Elipsa. I get:

#### Generator error
kfmon: error checking '/tmp/kfmon-ipc.ctl': stat: No such file or directory (src/generator_c.c:93)

This might be an error with kfmon instead? I don't know though. This is my first Kobo.

**Comment by NiLuJe** [comment]
Neither KFMon nor KOReader currently support the Elipsa.

NM itself ought to work, though.

**Comment by pgaskin** [comment]
> KOReader and kfmon are both independently broken on the new firmware

That's correct, except it's the device, not the firmware. Other software including Plato won't work either.

The NickelHook based ones will work fine.

> Is there a simpler program I should use to test

NM's arguably as simple as you can get (other than SSH/Telnet), tbh.

**Comment by mwt** [comment]
> NM's arguably as simple as you can get (other than SSH/Telnet), tbh.

I meant is there a simple script that I can launch with NM to test all of its actions/capabilities?

I get a KOReader log. So, it looks like **NM is successfully running the script I told it to run**. So, I think NM is working fine after all. I'm not sure if there's anything else to test on elipsis (eg. actions). If there is, I'm happy to try.

**Comment by NotAFile** [comment]
I am now encountering an identical error message on my Aura One as of version 4.33.19608
